docs/api-usage/developer-tools.mdx

```diff
@@ -9,11 +9,19 @@ Saleor includes a GraphQL Playground, an interactive GraphQL editor, allowing ac
 
 ![Playground](./api-playground.png)
 
-To view and try out the GraphQL Playground of Saleor's store, visit the [Saleor Storefront Example](https://storefront1.saleor.io/graphql/).
+#### Admin authenticated playgrounds
 
-Check [playground authentication](./authentication#authentication-with-playground) instructions to explore non-public APIs.
+In the admin dashboard use `CMD+'` this would open playground that would have admin rights of your respective user.
```

Comment:
What are the instructions for Windows/Linux users? 😄 

```suggestion
In the admin dashboard use `CMD+'`.  This will open playground that has admin rights of your respective user.
```

Reply:
it becomes Ctrl when it os not mac user, team covered that case